examples/ima.amp.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html ⚡>
+<head>
+    <meta charset="utf-8">
+    <title>IMA examples</title>
+    <link rel="canonical" href="amps.html" >
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+    <link href="//vjs.zencdn.net/5.3/video-js.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="http://gvabox.com/sbusolits/amp/css/videojs.ads.css" />
```

Comment:
Not sure how does this compare to Facebook. If your scripts already loads the stylesheets inside the iframe then there's no need to include them in the example page (or require the users to include them in their AMP docs).